src/software/SfM/main_ComputeFeatures.cpp

```diff
@@ -236,6 +257,12 @@ int main(int argc, char **argv)
         image_describer->Describe(imageGray, regions);
         image_describer->Save(regions.get(), sFeat, sDesc);
       }
+
```

Comment:
Since there is a continue sooner in the loop, this progress bar can have a unvalid value if one image cannot be open.
Will be better if we replace line 252 too
```
        Image<unsigned char> imageGray;
        if (ReadImage(sView_filename.c_str(), &imageGray))
        {
          // Compute features and descriptors and export them to files
          std::unique_ptr<Regions> regions;
          image_describer->Describe(imageGray, regions);
          image_describer->Save(regions.get(), sFeat, sDesc);
        }
```